src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -816,7 +816,8 @@ private void selectLessThanIv() {
         System.out.println("Selecting Pokemon with IV less than: " + ivTransfer.getText());
 
         for (int i = 0; i < pt.getRowCount(); i++) {
-            final double pIv = Double.parseDouble((String) pt.getValueAt(i, 3));
+            final double pIv = (double) pt.getValueAt(i, 5) * 100;
```

Comment:
Yep, I understand. I didn't take a hard look at the newer code, I just knew the selecting wasn't working when casting the getValueAt return object to a String for whatever reason. I will update it to use the enum